reconcile/utils/terrascript/cloudflare_resources.py

```diff
@@ -70,12 +72,15 @@ def populate(self) -> list[Union[Resource, Output]]:
                 f"for file path {gh_path} at ref {gh_ref}"
             )
         wrk_content = content.decode(encoding="utf-8")
+        tmp_file = tempfile.NamedTemporaryFile(delete=False)
```

Comment:
Maybe leverage `prefix` to give us better hints of what caused the creation of this if resources were ever leaked?

Reply:
I have refactored these changes to use a terraform variable instead of a temporary file. PTAL again 🙏🏻